config/_files.yml

```diff
@@ -1,338 +1,323 @@
 bootstrap:
```

Comment:
this file can be removed I think, it was needed when the files were hosted in the repo itself. Now both cdn/ dir and this file can be removed I believe a new one can be generated during build?

Reply:
So this files is generated with a new generation script that creates cdn from jsDeliver api. And then later app reads from the file to generate sri and display links on the website.

Reply:
Yes so we can remove it from the repo and run it during the build process and generate the file then?

Reply:
Yes, I just need to make a small change because the script now requires this file to exist before running the script, but I'll change that to create it on the fly.